cfme/tests/automate/test_domain.py

```diff
@@ -291,3 +294,68 @@ def test_object_attribute_type_in_automate_schedule(appliance):
         view.flash.assert_no_error()
         view.form.object_type.select_by_visible_text('<Choose>')
         view.flash.assert_no_error()
+
+
+@pytest.mark.tier(3)
+def test_copy_to_domain(domain):
+    """This test case checks whether automate class, instance and method are successfully copying to
+    domain.
+
+    Polarion:
+        assignee: ghubale
+        casecomponent: Automate
+        caseimportance: low
+        initialEstimate: 1/15h
+        startsin: 5.10
+        tags: automate
+        setup:
+            1. Create new custom domain
+        testSteps:
+            1. Go to Automation > Automate > Explorer
+            2. Select any class, instance and method from ManageIQ domain
+            3. Copy selected things one by one to new custom domain by selecting
+               "Copy this Method/Instance/Class" from configuration toolbar
+        expectedResults:
+            1.
+            2.
+            3. Class, Instance and Method should be copied to new domain and assert message should
+               appear after copying these things to new domain.
+
+    Bugzilla:
+        1500956
+    """
+    # Instantiating default domain - 'ManageIQ'
+    miq = (
+        domain.appliance.collections.domains.instantiate("ManageIQ")
+        .namespaces.instantiate("System")
+        .namespaces.instantiate("CommonMethods")
+    )
+
+    # Instantiating Class - 'MiqAe' from 'ManageIQ' domain
+    original_klass = miq.classes.instantiate("MiqAe")
+
+    # Copy this Class to custom domain
+    original_klass.copy_to(domain=domain)
+    klass = domain.browser.create_view(ClassCopyView)
+    klass.flash.wait_displayed()
+    klass.flash.assert_message("Copy selected Automate Class was saved")
+
+    # Instantiating Instance - 'quota_source' from 'ManageIQ' domain
+    original_instance = miq.classes.instantiate("QuotaMethods").instances.instantiate(
```

Comment:
Optional: You're using `miq.classes.instantiate("QuotaMethods")` more than once here, you could put it in a variable and then use it. I don't prefer this a lot, but we have been practicing this in our codebase.